frontend/Auth/Login.tsx

```diff
@@ -1,9 +1,165 @@
+      },
+    }
+  );
 
-export function Login(props) {
   return (
-    <Form method="post" action="/login">
-      <Button type="submit">Login</Button>
-    </Form>
+    <div style={{ display: "flex", justifyContent: "center" }}>
+      <Card
+        style={{
+          padding: "10px",
+          width: "30%",
+          boxShadow: "none",
+          marginTop: "15%",
+        }}
+      >
+        <h1 style={{ color: "#646461", fontSize: "20px" }}>Sign in</h1>
+        <Form onSubmit={unary(submitMutation.mutate)}>
+          <Form.Field>
+            <label htmlFor="username">Email</label>
+            <input
+              id="username"
+              value={loginFormData.username}
+              onChange={(evt) =>
+                setLoginFormData({
+                  ...loginFormData,
+                  username: evt.target.value,
+                })
+              }
+              required
+            />
+          </Form.Field>
+
+          <Form.Field>
+            <label htmlFor="password">Password</label>
+            <input
+              id="password"
+              value={loginFormData.password}
+              onChange={(evt) =>
+                setLoginFormData({
+                  ...loginFormData,
+                  password: evt.target.value,
+                })
+              }
+              required
+            />
+          </Form.Field>
+
+          <Button
+            type="submit"
+            style={{
+              background: "#2a467b",
+              color: "white",
+              width: "50%",
+              margin: 0,
+            }}
+          >
+            Sign in
+          </Button>
+
+          <Button
+            type="submit"
+            style={{
+              background: "white",
+              color: "#2a467b",
+              width: "50%",
+              margin: 0,
+            }}
+          >
+            Reset Password
+          </Button>
+
+          <div style={{ marginTop: "20px", marginBottom: "10px" }}>
+            <div
+              style={{
+                width: "100%",
+                height: "10px",
+                textAlign: "center",
+                borderBottom: "1px solid #b3b3b1",
+                color: "#b3b3b1",
+              }}
+            >
+              <span
+                style={{
+                  fontSize: "10px",
+                  backgroundColor: "white",
+                  padding: "0 10px",
+                }}
+              >
+                or
+              </span>
+            </div>
+          </div>
+
+          <Button
+            class="ui grey basic button"
+            style={{ marginTop: "20px", width: "100%", borderColor: "red" }}
+          >
+            Insert Google Button here
+          </Button>
+
+          <p
+            style={{ textAlign: "center", fontSize: "12px", marginTop: "30px" }}
+          >
+            New to Flax?{" "}
+            <a href="/create-account" style={{ textDecoration: "underline" }}>
```

Comment:
We should use react-router's `<Link>` element here to avoid a page refresh when navigating:

https://reactrouter.com/web/api/Link